roles/ceph-rgw/tasks/multisite/secondary.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: ensure connection to primary cluster
+  uri:
+    url: "{{ item.endpoint }}"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
```

Comment:
Do we need to test the connection to the primary cluster from the monitor ? or it should be done from the rgw nodes ?

Reply:
Yes since that's the place where the radosgw-admin realm pull command will be run from.